botometerAnalyser/routes.js

```diff
@@ -16,22 +20,10 @@ if (process.env.NODE_ENV !== 'test') {
 	});
 }
 
-router.get('/', async (req, res, next) => {
-	const { token: givenToken, text: search } = req.query;
-
-	if (givenToken !== mattermostToken) {
-		return res.status(401).json({ Error: 'Missing or invalid token' });
-	}
-
-	if (!search) {
-		return res.json({
-			text: 'Hey! I can help you by analyzing the latest shares on a specific topic but I need a query.\nSo, give me a keyword, an URL or some text. For example: \n`/botometer disinformation`'
-		});
-	}
-
-	const activeJobsCount = await queue.getActiveCount();
+async function startQueryTextPipeline({ req, res, search }) {
```

Comment:
I think it's better to avoid passing `req` and `res` as params, it makes testing more difficult. 
Pass only required params from `req.query` and return the response text.
We can discuss about this.

Reply:
done